accounts/abi/bind/backends/simulated.go

```diff
@@ -779,6 +779,7 @@ func (m callMsg) GasPrice() *uint256.Int         { return m.CallMsg.GasPrice }
 func (m callMsg) FeeCap() *uint256.Int           { return m.CallMsg.FeeCap }
 func (m callMsg) Tip() *uint256.Int              { return m.CallMsg.Tip }
 func (m callMsg) Gas() uint64                    { return m.CallMsg.Gas }
+func (m callMsg) DataGas() uint64                { return params.DataGasPerBlob * uint64(len(m.CallMsg.DataHashes)) }
```

Comment:
Can you help me understand why DataGas = DataGasPerBlob * len(datahashes), shouldn't it be blob numbers?

Reply:
Uhh, is it because that there'll be only 1 bob, and inside that data blob, we only store data hashes?

Reply:
Yes it is # of blobs... and for each blob, there is 1 hash in DataHashes.  Thus len(DataHashes) gives you the # of blobs.